programs/programs/urgent_needs/il/housing_development_authority_foreclosure_prevention_counseling.py

```diff
@@ -0,0 +1,14 @@
+from ..base import UrgentNeedFunction
+
+
+class HousingDevelopmentAuthorityForeclosurePreventionCounseling(UrgentNeedFunction):
+
+    def eligible(self):
+        """
+        Return True if the household marks need for housing/utilities resource and household has a mortgage expense
+        """
+
+        needs_housing_help = self.screen.needs_housing_help
```

Comment:
This works, but I would suggest to remove this condition and instead configure this with "type short" in django admin.
<img width="1744" height="538" alt="image" src="https://github.com/user-attachments/assets/ea344d9d-7cd7-411d-8f9d-c5ea4990fea8" />

Reply:
Curious, is there a specific benefit to configuring only the "type short" in Django? I went with both, since I feel that keeping both conditions makes the eligibility requirements easier to understand.